src/pages/Providers/utils/handleProvidersList.ts

```diff
@@ -0,0 +1,27 @@
+import { getListOfProviders } from 'src/actions';
+import { Provider } from 'src/utils';
+import { ref } from 'vue';
+
+export const handleProvidersList = () => {
+  const isLoading = ref<boolean>(false);
+  const providers = ref<Provider[]>([]);
+
+  const getProvidersList = async (_limit?: number) => {
```

Comment:
why not just limit? then in line 13 would be just 

```js
    const variables = {
      limit,
    };
```

Reply:
Thanks for the suggestion, will apply